rebar.config

```diff
@@ -97,7 +97,9 @@
   % test case, but leave it up to the tests to actually create the directories.
   {create_priv_dir, manual_per_tc},
   % Specify that we want the results captured in a Surefire report.
-  {ct_hooks, [{cth_surefire, [{path, "../report.xml"}]}]}
+  {ct_hooks, [{cth_surefire, [{path, "../report.xml"}]}]},
+  % Config for tests
+  {sys_config, ["test/test.config"]}
```

Comment:
Because I see that in CT we have a lot of places where we use the same values like `{'MS-Primary-DNS-Server', {8,8,8,8}}` and others, this is a simple copy and paste, the firs point why I added is - do set configuration for TEID only once and second point is also if we will be have some immutable configuration we can set configuration only once without putting the same one config in a lot of places in CTs. But if you think that this is bad approach I can remove it.

Reply:
Yes. Please, remove that config file.